### DISCUSSION TITLE
Google Colab - colab.research

### DISCUSSION THREAD

**Discussion by AndrewStatistician** [rramosp/ai4eng.v1#39] [00 - General]
Buenos días,  Soy Andres Castro y estoy realizando el curso.

No había tenido problema al cargar los códigos .ipynb hasta ayer que abrí el archivo, guarde una copia en mi drive pero al abrirlo directamente en mi drive no fue posible, aparece el siguiente mensaje:

![image](https://user-images.githubusercontent.com/52263608/123643784-5eef6b00-d7ea-11eb-82c5-df06989cf642.png)

Error de carga del bloc de notas
Se produjo un error al cargar este bloc de notas. Asegúrate de que se pueda acceder al archivo y vuelve a intentarlo.
Failed to fetch

https://drive.google.com/drive/?action=locate&id=1LxVJmu8g-f7ynxR7sCvmlZzS150D5y-_&authuser=0
Failed to fetch
TypeError: Failed to fetch

Sera algún problema de Google drive del pc? han tenido algún caso similar? que puedo hacer en este caso?

Muchas gracias por la ayuda.

**Comment by Felipe-RA** [comment]
Saludos Andrew

Este problema puede tener muchas causas, las más comunes es problemas para cargar las cookies que utiliza colab y que el archivo no sea accesible desde colab (error de escritura, permisos)

Para el primero buscate cómo permitir las cookies de terceros en tu navegador, o haz una excepción sólo para colab y drive.
Para el segundo usualmente la solución está en crear otro archivo nuevo o abrirlo en local y ver qué puede recuperarse en caso que tengas avance que no quieras perder.

**Reply by AndrewStatistician** [reply]
Muchas Gracias Felipe, ya pude abrir los archivos. 👯
